lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
will have to read closer, but don't think this will work. this unconditionally causes appmetrics to be required. appmetrics require has side effects, but strong-supervisor can run with a `--no-profile` option, name is historical, but it means no monitoring code is loaded - useful to determine if bugs in apps are caused by side effects of monitoring. we can talk later if its too hard to figure out the expectations here.